crates/cli/tests/reference/enums.js

```diff
@@ -78,9 +79,16 @@ Yellow:1,"1":"Yellow",
  * Red as a rose.
  */
 Red:2,"2":"Red", });
-
+/**
+ * @enum {0 | 1 | 42 | 43}
+ */
 export const ImplicitDiscriminant = Object.freeze({ A:0,"0":"A",B:1,"1":"B",C:42,"42":"C",D:43,"43":"D", });
 
+/**
+ * The name of a color.
```

Comment:
Considering this isn't exported, I believe doc comments shouldn't be exposed as well.
WDYT?

Reply:
I think that's reasonable. I'll remove them.